leakcanary-analyzer/src/main/java/com/squareup/leakcanary/AnalysisResult.java

```diff
@@ -20,16 +20,17 @@
 public final class AnalysisResult implements Serializable {
 
   public static AnalysisResult noLeak(long analysisDurationMs) {
-    return new AnalysisResult(false, false, null, null, null, analysisDurationMs);
+    return new AnalysisResult(false, false, null, null, null, 0, analysisDurationMs);
```

Comment:
dang. Normally I'm the one writing this sort of comment on PRs. Oh well. I'd argue this is "ok ish" because `AnalysisResult` has a private constructor and a small surface, so it's all in one place.

I could also extract local variables.